readme.md

```diff
@@ -102,10 +108,14 @@ try {
 
 ## API
 
-### execa(file, [arguments], [options])
+### execa(command, [arguments], [options])
```

Comment:
I don't think we should change this to command as it still support file, which is what I still would recommend to people.
```suggestion
### execa(file | command, [arguments], [options])
```

Happy to consider other options.